### DISCUSSION TITLE
Unclear part for building on Windows with MSYS2

### DISCUSSION THREAD

**Discussion by LuciferSam86** [FluidSynth/fluidsynth#1132] [Q&A]
Hello there,
I was looking the build instructions on msys2 for windows and I saw this

> Note: Make sure to execute fluidsynth in the MSYS2 shell as well. It wont work form the cmd.exe (out-of-the-box).

What does that mean? And how could make possible to use fluidsynth outside MSYS2?

Thank you!

**Comment by ghost** [comment]
You can try appending the binary directory of fluidsynth such as `C:/fluidsynth/bin` into the `%Path%` environment variable so that you can use `fluidsynth.exe` outside msys2.
To append this directory into `%PATH%`, open control panel first. Then enter `env` in the search box. Click `Edit system environment variables` and click `environment variables`. You will find `Path` in the popup window.
